### PR TITLE
Style header footer

### PR DESCRIPTION
**Pull request by mikeyrayvon**

### CODE REVIEW

File: partials/logo.php [hunk 1]
```diff
@@ -0,0 +1 @@
....7 1.7v2.7zm53.6-9.6h4.9v28.6h-4.9zm-184.6 4.7h4.7v23.9h4.9V14.5h4.7V9.8h-14.3zm61.8 23.9h12.8v-4.9h-7.9v-7.2h6.9v-4.7h-6.9v-7.1h7.9V9.8h-12.8zM78.4 32.5h4.3l1.1 5.9H89L82.7 9.8h-4.2l-6.4 28.6h5.2l1.1-5.9zm2.2-13.8l1.7 9.1H79l1.6-9.1zm296.3 13.8h4.3l1.1 5.9h5.2l-6.4-28.6h-4.2l-6.4 28.6h5.2l1.2-5.9zm2.1-13.8l1.7 9.1h-3.3l1.6-9.1zm61 14.8V9.8h-5v28.6h12.8v-4.9zm122.1-7l-2.7-16.6h-5.2l5.8 28.4h4.2l5.9-28.4h-5.2zm113.7 11.8h6.8c2 0 3.8-.8 5.2-2.3 1.2-1.4 1.8-3.1 1.8-5V16.7c0-2-.7-3.7-2.3-5.1-1.3-1.2-2.9-1.8-4.7-1.8h-6.7v28.5zm4.9-23.8h1.5c.7 0 1.2.2 1.6.6.5.4.7 1 .7 1.6V31c0 .9-.2 1.6-.6 2-.4.4-1 .6-1.9.6h-1.4V14.5zm-56.1-4.2c-2-1-4.4-.9-6.3.1-1 .5-1.9 1.3-2.5 2.2-.9 1.2-1.3 2.7-1.3 4.4v14.1c0 1.7.4 3.2 1.3 4.4.7.9 1.5 1.7 2.6 2.2 1 .5 2.1.8 3.3.8 1.1 0 2.1-.2 3.1-.7 1-.5 1.8-1.1 2.5-1.9 1-1.3 1.6-2.9 1.6-4.8V17c0-1.9-.5-3.5-1.6-4.8-.8-.8-1.7-1.4-2.7-1.9zm-.8 20.7c0 .9-.1 1.3-.3 1.5-.5.8-1.1 1.1-2 1.1-.7 0-1.3-.3-1.8-1-.3-.4-.4-.9-.4-1.7V17c0-.6.1-1 .3-1.3.5-.8 1.1-1.2 2-1.2.7 0 1.3.3 1.8.9.3.3.4.9.4 1.6v14zm127.3 7.3l-6.5-17 5.6-11.4h-5l-5.1 10.9V9.9h-4.9v28.4h4.9v-8.5l1.6-3 4.1 11.5zm48.2-5.8h4.3l1.1 5.9h5.2l-6.4-28.6h-4.2L793 38.4h5.2l1.1-5.9zm2.2-13.8l1.7 9.1h-3.3l1.6-9.1z"/></svg>
```
Comment (varffs) [inline partials/logo.php:1]: How come we went back to this `.svg` approach? I still prefer the include.

File: partials/seo.php [hunk 2]
```diff
@@ -29,7 +29,7 @@
 } else if (!empty($ogImage)) {
   echo '<meta property="og:image" content="' . $ogImage[0] . '" />';
 } else {
-  echo '<meta property="og:image" content="' . get_stylesheet_directory_uri() . '/img/dist/favicon.png" />';
+  echo '<meta property="og:image" content="' . get_template_directory_uri() . '/dist/img/favicon.png" />';
```
Comment (varffs) [inline partials/seo.php:32]: Is this wrong in the boiler?

File: src/js/main.js [hunk 3]
```diff
@@ -42,12 +48,38 @@ class Site {
...
-
-    if (_this.$covervidVideo.length) {
-      _this.$covervidVideo.coverVid(1920, 1080);
+    if (this.$covervidVideo.length) {
+      this.$covervidVideo.coverVid(1920, 1080);
     }
   }
+
+  bindStickyHeader() {
+    const that = this;
```
Comment (varffs) [inline src/js/main.js]: Rather than `_this`? I can't remember what differences there are now with ES6

**Review by varffs — APPROVED** [review]
Obviously fine but a few questions on conventions

File: src/js/main.js [hunk 4]
```diff
@@ -42,12 +48,38 @@ class Site {
...
+
+    this.sizeHeaderSpacer();
+
+    this.$window.on('scroll', function() {
```
Comment (mixcas) [inline src/js/main.js]: The function that handles the scroll should be a different function (ex. `Site.handleOnScroll()`) that in the `constructor` is bound to `this` (ex. `this.handleOnScroll = this.handleOnScroll.bind(this)`. The same way we do on our React code.

Comment (mixcas) [reply src/js/main.js]: related discussion about this new class pattern https://github.com/facebook/react/issues/9851

File: partials/logo.php (hunk 1 above)
Comment (mikeyrayvon) [reply partials/logo.php:1]: because it never works

File: partials/seo.php (hunk 2 above)
Comment (mikeyrayvon) [reply partials/seo.php:32]: yes. the whole seo partial is pretty wrong

File: src/js/main.js (hunk 3 above)
Comment (mikeyrayvon) [reply src/js/main.js]: yea just trying it on for size

File: partials/logo.php (hunk 1 above)
Comment (varffs) [reply partials/logo.php:1]: `never` is pretty inaccurate

**Pull request merged**